README.md

```diff
@@ -1,13 +1,13 @@
 # Mongo API Project
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This is a project as part of Technigo Bootcamp. This weeks assignment was to connect a database using mongoDB to our built API.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I had a lot of errors when trying to connect to mongodb compass and since i've never used it before it was harder to understand the error messages but finally I was able to do it. I used the books data which were provided by Technigo to build a few end points and store the data in mongoDB.
```

Comment:
It would be nice to read more about how you planned your project and what tools and techniques you used in your README. 🙂